### PR TITLE
Logs in tests are enabled/disabled now

### PR DESCRIPTION
**Pull request by outsideris**
### Github Issue
Related #111 

### Background
Logs are too verbose during running tests.
To digging #111, this feature is needed

### Solution
`SEBAK_LOG_HANDLER` env var can enable or disable logger.

This PR is made by pair(triple?) programming with @anarcher and @zzim2x

### CODE REVIEW

File: lib/common/test/log.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	logging "github.com/inconshreveable/log15"
+)
+
+func LogHandler() logging.Handler {
+	handlers := map[string]func() logging.Handler{
+		"null": func() logging.Handler {
+			return logging.DiscardHandler()
+		},
+		"stdout": func() logging.Handler {
+			return logging.StreamHandler(os.Stdout, logging.TerminalFormat())
```
Comment (Geod24) [inline lib/common/test/log.go]: Use `StdoutHandler` instead

Comment (anarcher) [reply lib/common/test/log.go]: @Geod24 @outsideris 

```
  var  StdoutHandler = StreamHandler(os.Stdout, LogfmtFormat())
```
https://godoc.org/github.com/inconshreveable/log15#pkg-variables

I think `TerminalFormat` is better than `LogfmtFormat` in console :-D

Comment (Geod24) [reply lib/common/test/log.go]: It is actually a `TerminalFormat` with support for color, but it's [initialized at runtime](https://github.com/inconshreveable/log15/blob/d4e174bf71609b5afbd4a559f7477f370450053d/root.go#L17-L20).
I've replaced similar code with `StdoutHandler` in #94

**Comment by outsideris** [comment]
@Geod24 I fixed it.

**Comment by astonshnam** [comment]
I reviewed it now because I did a different job. This looks very good to me.

**Pull request merged**